examples/panzoom_camera.py

```diff
@@ -0,0 +1,84 @@
+"""
+Example showing orbit camera controls.
+"""
+
+import numpy as np
+import imageio
+import pygfx as gfx
+
+from PyQt5 import QtWidgets, QtCore
+from wgpu.gui.qt import WgpuCanvas
+
+
+class WgpuCanvasWithInputEvents(WgpuCanvas):
+    _drag_modes = {QtCore.Qt.RightButton: "pan"}
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.drag = None
+
+    def wheelEvent(self, event):  # noqa: N802
+        dim = self.width(), self.height()
+        pos = event.x(), event.y()
+        delta = event.angleDelta().y() * 0.00125
+        view = camera.right - camera.left, camera.top - camera.bottom
+        controls.zoom_to_point(delta, pos, dim, view)
```

Comment:
If we provide the control a ref to the camera (instead of only its initial position), then we e.g. would not have to provide the view here ...